core/src/main/java/org/jdbi/v3/core/argument/Arguments.java

```diff
@@ -60,6 +63,22 @@ public Arguments register(ArgumentFactory factory) {
                 .findFirst();
     }
 
+    /**
+     * Configure the {@link Argument} to use when binding a null
+     * we don't have a type for.  Intended for vendor plugins to call.
```

Comment:
This latter sentence, the way it's worded, may discourage users from configuring this by hand if needed. e.g. there is no `JdbiPlugin` for their database vendor, or the plugin does not configure this.